src/Symfony/Component/Yaml/Inline.php

```diff
@@ -564,6 +565,8 @@ private static function evaluateScalar($scalar, $flags, $references = array())
                         return;
                     case 0 === strpos($scalar, '!!float '):
                         return (float) substr($scalar, 8);
+                    case preg_match('{^[+-]?[0-9][0-9_]*$}', $scalar):
+                        $scalar = str_replace('_', '', (string) $scalar);
```

Comment:
Please add a comment saying that the fallthrough is intentional, to avoid mistakes in the future